src/VisualStudio/Core/Def/Implementation/Utilities/BrowserHelper.cs

```diff
@@ -59,7 +59,8 @@ private static bool TryStartBrowser(IServiceProvider serviceProvider, Uri uri)
 
         private static bool TryStartBrowser(IVsWebBrowsingService service, Uri uri)
         {
-            return ErrorHandler.Succeeded(service.CreateExternalWebBrowser((uint)__VSCREATEWEBBROWSER.VSCWB_ForceNew, VSPREVIEWRESOLUTION.PR_Default, uri.AbsoluteUri));
+            IVsWindowFrame unused;
+            return ErrorHandler.Succeeded(service.Navigate(uri.AbsoluteUri, 0, out unused));
         }
 
         private static void StartBrowser(Uri uri)
```

Comment:
@shyamnamboodiripad why not? my opinion is consistency is good. but worst is it silently failing and do nothing only because it couldn't keep consistency. as bad as we crashing VS, because outlining couldn't figure out a region.